nlp/2021_07_07_neural_keyword_extraction/README.md

```diff
@@ -0,0 +1,11 @@
+# Neural Keyword Extraction
+
+Advancements in neural networks are applied to an ever growing number of natural language processing problems. Keyword extraction is no exception. The emerging subcategory, dubbed ***neural* keyword extraction** 🧠, often uses sequence-to-sequence models, whose objective is to transform a sequence of input words into an abstract intermediate representation and from it generate a sequence of keywords.
+
+This is in contrast to classical keyword extraction which is often graph-based and utilises linguistic 💬 or statistical 📊 features.
+
+💡 In this quick-tip we look at two approaches that fall into the category of neural keyword extraction.
+
+We recommend to open the notebook using Colab for an interactive explainable experience and optimal rendering of the visuals 👇:
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ml6team/quick-tips/blob/main/nlp/2021_07_07_neural_keyword_extraction/neural_keyword_extraction.ipynb)
```

Comment:
Link didn't work for me

Reply:
That's expected as the pull request is not yet merged and thus the notebook is not part of the main branch. :)

Reply:
placeholder link for PR colab:  https://colab.research.google.com/github/ml6team/quick-tips/blob/feature/nlp/neural-keyword-extraction/nlp/2021_07_07_neural_keyword_extraction/neural_keyword_extraction.ipynb